### ISSUE TITLE
Kaizou Choujin Shubibinman Zero not working

### ISSUE THREAD

**Issue by Sun-in-jang** [snes9xgit/snes9x#388]
As title says, when you launch this game nothing happens - black screen and no sound. Game ROM-file name is messy too (on screenshot). Also this game probably require BIOS - i have "BS-X BIOS" in BIOS folder (Snes9x - BIOS), and even with it game doesn't work (File - Load Game...)

On bsnes, this game doesn't require bios, there is option "Expansion port device" (Satellaview).
![snes9x](https://user-images.githubusercontent.com/38378342/44031495-dbef0634-9f0c-11e8-80e7-b20c0d100284.PNG)

**Comment by retrotalker** [comment]
open snes9x.exe.

load multicart.
- slot a: BS-X - Sore wa Namae o Nusumareta Machi no Monogatari (Japan) (Rev 1).sfc
- slot b: Kaizou Choujin Shubibinman Zero (Japan).sfc

no-intro 2018
MD5: DB4DD0305051F7D3CA7BD2E114F4F1DD
SHA-1: DD084021A7E2AFD939FEE3B6426778F6E5C2F333

skip bios intro. enter house. load stored data. choose game. play.



As for bsnes-plus, I went to advanced - expansion port device - satellaview. load special - load bs-x cartridge. went through bios and played game.

Tried load cartridge directly with bs game but nothing happened. Black screen like snes9x.
